NOTES.WIN

```diff
@@ -83,6 +83,18 @@
  is, of course, to choose a different set of directories by using
  --prefix and --openssldir when configuring.
 
+
+ Special notes for Universal Windows Platform builds, a.k.a. VC-*-UWP
+ --------------------------------------------------------------------
+
+ - UWP targets only support building the static and dynamic libraries.
+
+ - The "no-uplink" must be given in the "Configure" script.
```

Comment:
Is this actually necessary if we make the change I suggest above?

Reply:
I think I can remove the "The "no-uplink" must be given in the "Configure" script." in the note.

Reply:
I have removed "The "no-uplink" must be given in the "Configure" script." in the note.